assets/training/finetune_acft_image/environments/acpt_image_huggingface/context/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# PTCA image
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:{{latest-image-tag:vision\.biweekly\.\d{6}\.\d+}}
```

Comment:
aifx deprecated vision.biweekly, use biweekly tag

Reply:
@sheetalarkadam is helping address it. Will update by tomorrow noon PST

Reply:
The image is now available in the new namespace: mcr.microsoft.com/aifx/acpt/vision/stable-ubuntu2004-cu117-py38-torch1131:biweekly.202308.1

Reply:
@vizhur updated the base image and corresponding validation runs are tagged in description.